generated/nitclk/nitclk/session.py

```diff
@@ -303,18 +246,7 @@ def _get_attribute_vi_string(self, attribute_id):
             value (str): The value that you are getting
 
         '''
-        session_ctype = _visatype.ViSession(self._session_number)  # case S110
-        channel_name_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
-        attribute_id_ctype = _visatype.ViAttr(attribute_id)  # case S150
-        buf_size_ctype = _visatype.ViInt32()  # case S170
-        value_ctype = None  # case C050
-        error_code = self._library.niTClk_GetAttributeViString(session_ctype, channel_name_ctype, attribute_id_ctype, buf_size_ctype, value_ctype)
-        errors.handle_error(self, error_code, ignore_warnings=True, is_error_handling=False)
-        buf_size_ctype = _visatype.ViInt32(error_code)  # case S180
-        value_ctype = (_visatype.ViChar * buf_size_ctype.value)()  # case C060
-        error_code = self._library.niTClk_GetAttributeViString(session_ctype, channel_name_ctype, attribute_id_ctype, buf_size_ctype, value_ctype)
-        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
-        return value_ctype.value.decode(self._encoding)
+        return self._library_interpreter.get_attribute_vi_string(self._repeated_capability, attribute_id)
 
     def _get_extended_error_info(self):
```

Comment:
Should we do something to prevent this from generating ins session.py? It's not called by anything.

Reply:
Stopped it from generating in session.py.